<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CENTAUR_NAMESPACE::QRCodeDialog</class>
 <widget class="QDialog" name="CENTAUR_NAMESPACE::QRCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:  white;
border: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}

QLabel{
font-size:  14px;
}
QFrame{
border: 0px;
}

QLabel#codeLabel{

	font: 700 18pt &quot;Times New Roman&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>65555</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="qrCodeLabel">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>156</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>156</width>
             <height>156</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	background-color: black;
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Scan this QR code in the Google Authenticator app</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>50</number>
      </property>
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>156</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: black;
}</string>
           </property>
           <property name="text">
            <string>If you are unable to scan this QR code, please enter this code in the app manually</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="codeLabel">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: black;
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="closeLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: blue;
}</string>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;http://invalid-url.com/&quot;&gt;Close&lt;/a&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextBrowserInteraction</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
